infrastructure/disk.tf

```diff
@@ -92,6 +106,13 @@ resource "aws_s3_bucket" "data_refinery_transcriptome_index_bucket" {
   }
 }
 
+resource "aws_s3_bucket_public_access_block" "data_refinery_transcriptome_index_bucket" {
+  bucket = "${aws_s3_bucket.data_refinery_transcriptome_index_bucket.id}"
+
+  block_public_acls   = true
+  block_public_policy = true
+}
```

Comment:
This bucket does not need to be blocked. (See https://github.com/AlexsLemonade/refinebio/issues/731#issuecomment-469443058)